rfcs/20210518-prelu-postoperation/README.md

```diff
@@ -0,0 +1,55 @@
+# Introducing PReLU post operation support in oneDNN
+
+## Motivation
+
+This paper https://arxiv.org/abs/1502.01852 introduced PReLU activation
+function. The DIEN model uses PReLu as matmul activation.
+This gets expanded in TF graph.
+![](graph_prelu.png "Graph attention layer")
+
+It becomes a bottleneck for the Attention layer in DIEN. Currently oneDNN
```

Comment:
What is expected performance improvement?

Reply:
For know I don't have exact values. I'm trying for several days reach requestor and get answers about more details. I will go back to you as soon as possible.

Reply:
Framework team is optimizing  FSRCNN on GPU from a customer. The network is mainly composed of conv + prelu pattern.

Now: 8.7ms(conv ) + 4.5ms(prelu, not fused) = 13.2ms
Expectation: 8.7ms(conv) + 0.4ms(10x perf gain by fusing) = 9.1ms

Reply:
Generally expected performance improvement should be similar to fusing binary postops, here main cost will be loading weights, main advantage will be the presence prelu src tensor in registers.

Reply:
@igorsafo Do you need more details or can the the thread be resolved?